app/src/main/java/com/zulip/android/BuildHelper.java

```diff
@@ -0,0 +1,18 @@
+package com.zulip.android;
+
+import android.os.Build;
+
+/**
+ * Class for build logic
+ * Created by lisaneigut on 24 Mar 2016.
```

Comment:
You can omit this line, since attribution and datestamps will be handled by `git log` / `git blame`.